contracts/multisig/src/multisig.rs

```diff
@@ -60,12 +58,33 @@ pub trait Multisig:
     /// - the serialized action data
     /// - (number of signers followed by) list of signer addresses.
     #[label("multisig-external-view")]
+    #[allow_multiple_var_args]
```

Comment:
This annotation isn't needed if you use two OptionalValue.

Reply:
`Invalid contract variant multisig-full: Multiple var args found in get_pending_action_full_info. Use #[allow_multiple_var_args] if you want to enable this feature`

the contract build disagrees with you